src/@sdk/react/components/CheckoutProvider/contextHandlers.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect } from "react";
+
+import { Checkout } from "@sdk/fragments/types/Checkout";
+
+import { useUpdateCheckoutLine } from "../..";
+import { ApolloErrorWithUserInput } from "../../types";
+import { ICheckoutContextStateHandlers } from "./context";
+import { ICartItem } from "./types";
+
+interface IContextStateHandlersParams {
+  checkout: Checkout | null;
+  updateCheckout: (checkout: Checkout | null) => void;
+  updateLoading: (loading: boolean) => void;
+  updateError: (error: ApolloErrorWithUserInput | null) => void;
+  updateShippingAsBilling: (shippingAsBilling: boolean) => void;
+}
+
+export const useCheckoutContextStateHandlers = ({
+  checkout,
+  updateCheckout,
+  updateLoading,
+  updateError,
+  updateShippingAsBilling,
+}: IContextStateHandlersParams): ICheckoutContextStateHandlers => {
+  const [
+    setUpdateCheckoutLine,
+    { data: updateData, loading: loadingData, error: errorData },
+  ] = useUpdateCheckoutLine();
+
+  useEffect(() => {
+    const updatedCheckout = updateData?.checkout;
+
+    if (updatedCheckout) {
```

Comment:
There is no need for temp variable - you can just `if(updateData?.checkout)`

Reply:
It is referenced also after if `updateCheckout(updatedCheckout)`

Reply:
ok ill change to -> updateCheckout(updateDate?.checkout || null)